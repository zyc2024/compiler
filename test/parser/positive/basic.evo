import K
import J
import L

int add5(int x) {
    return x+5;
}