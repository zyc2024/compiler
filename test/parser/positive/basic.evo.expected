( ((import K)(import J)(import L))
    (
        (add5 ((x int)) (int) ((return (+ x 5)))) 
    )
)